src/main/java/uk/gov/hmcts/reform/pip/account/management/database/UserRepository.java

```diff
@@ -24,10 +25,10 @@ List<PiUser> findExistingByProvenanceId(@Param("provUserId") String provenanceUs
 
     Optional<PiUser> findByUserId(UUID userId);
 
-    @Query(value = "SELECT cast(user_id as text), provenance_user_id, user_provenance, roles, created_date, "
-        + "last_signed_in_date FROM pi_user",
-        nativeQuery = true)
-    List<String> getAccManDataForMI();
+    @Query("SELECT new uk.gov.hmcts.reform.pip.model.report.AccountMiData("
+        + "u.userId, u.provenanceUserId, u.userProvenance, u.roles, u.createdDate, u.lastSignedInDate) "
+        + "FROM PiUser u")
+    List<AccountMiData> getAccManDataForMI();
```

Comment:
This is not your change but while you are there are you OK to rename the method name to something like getAccountDataForMi() or getMiData() or getAccountMiData(), etc.? 'AccMan' does not sound good :)

Reply:
done :)